lib/galaxy/web/form_builder.py

```diff
@@ -312,7 +313,11 @@ def __init__( self, name, multiple=None, display=None, refresh_on_change=False,
     def add_option( self, text, value, selected=False ):
         self.options.append( ( text, value, selected ) )
 
-    def get_html( self, prefix="", disabled=False ):
+    def get_html( self, prefix="", disabled=False, extra_attr=None ):
+        if extra_attr is not None:
+            self.extra_attributes = ' %s' % ' '.join( [ '%s="%s"' % ( k, v ) for k, v in extra_attr.items() ] )
```

Comment:
Can you attribute escape this (https://docs.python.org/2/library/xml.sax.utils.html#xml.sax.saxutils.quoteattr) - I guess it is only used in one place so it isn't currently exposing a vulnerability but I'd like other potential consumers of this to have their values automatically escaped - at least by default.